src/Raygun4php/RaygunClient.php

```diff
@@ -68,6 +68,22 @@ public function __construct($key, $useAsyncSending = true, $debug = false, $disa
       $this->disableUserTracking = $disableUserTracking;
     }
 
+    /**
+    * @param bool $disableUserTracking True to disable user tracking
```

Comment:
The indentation of this docblock is one space out =)

Also might be worth making this setter fluent. You could also use it in the constructor for free code coverage.